capstone/capweb/templates/api.html

```diff
@@ -34,13 +34,18 @@ <h1>
         <nav class="list-group sidebar-menu" aria-label="table of contents">
           <ul>
             <li><a class="list-group-item" href="#overview">Overview</a></li>
-            <li><a class="list-group-item" href="#registration">Registration</a></li>
             <li><a class="list-group-item" href="#usage">Using CAPAPI</a></li>
-            <li><a class="list-group-item" href="#beginners">Beginner's Introduction</a></li>
+            <ul>
```

Comment:
HTML cop: only `<li>` can be children of `<ul>`. To nest a list inside another list:
```
<ul>
    <li>
        <ul>
            <li></li>
            <li></li>
        </ul>
   </li>
   <li></li>
   <li></li>
</ul>
```